.sass-lint.yml

```diff
@@ -53,9 +53,7 @@ rules:
     -
       style: 'short'
   hex-notation:
-    - 1
-    -
-      style: 'uppercase'
+    - 0
```

Comment:
You should make a PR on the tools repository (https://github.com/Talend/tools/blob/master/tools-scss-formatting/.sass-lint.yml) prior to this change.

After that, please, make another PR for this change.

Reply:
This is be aligned with perttier ? In that case I auto agree with the PR on tools :D